### PR TITLE
About us page

### LINKED ISSUE #24: About Us Page
image

### PR DESCRIPTION
**Pull request by ayse8888**
Hi,
I built About Us Page
@halitbatur here I have some notes about the page.
1. In the header section, right / left works on Mustafa's side but does not work on my side when i change left to right. We tried more than 1 hour but could not make it work. Not to lose any more time since we have many missing parts , I chose another picture here for left side as attached.
image

2. In the next section, I mentioned you I am not able to change the modal content completely since it is coming from a merged component and unfortunately I used partial props there. And not to break any code, I did not change the data from the merged component but if that's ok, I can change. The thing I wanted to put was some articles here but I had to change it as "latest adoptions" to make it fit to the structure.
image
image

3. I changed some little css from a merged component when i am in about-us-page branch as per my conversation with you since there was duplicate class names.

4. And finally the team part is below
image

### CODE REVIEW

File: src/pages/aboutus/teamdata.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from "react";
```
Comment (halitbatur) [inline src/pages/aboutus/teamdata.js]: can we move this to data folder ?

File: src/pages/aboutus/worksdata.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import poodle from "../../images/poodle.jpg";
```
Comment (halitbatur) [inline src/pages/aboutus/worksdata.js]: can we move this to a data folder ?

**Review by halitbatur — APPROVED** [review]

**Review by halitbatur — CHANGES_REQUESTED** [review]
some few changes and then its ready

File: src/pages/aboutus/teamdata.js (hunk 1 above)
Comment (ayse8888) [reply src/pages/aboutus/teamdata.js]: done

File: src/pages/aboutus/worksdata.js (hunk 2 above)
Comment (ayse8888) [reply src/pages/aboutus/worksdata.js]: done

**Review by halitbatur — APPROVED** [review]
Looks good to me Ayse good work !

**Comment by halitbatur** [comment]
@ayse8888 could u updated this branch and resolve conflicts ?

**Review by mustafaguldag — APPROVED** [review]
Hi Ayşe, it seems nice to me. There is no unnesesary spaces. Thank you for your contribute!

**Pull request merged**